agent/src/beerocks/slave/backhaul_manager/backhaul_manager_thread.h

```diff
@@ -519,6 +520,15 @@ class backhaul_manager : public btl::transport_socket_thread {
                           const sLinkNeighbor &link_neighbor, const sLinkMetrics &link_metrics,
                           ieee1905_1::eLinkMetricsType link_metrics_type);
 
+    struct sExpectedApMetricsResponse {
```

Comment:
```suggestion
    struct sApMetricsResponse {
```
I think we should reconsider the name for this struct, though I'm not happy with this name either. Suggestions are welcome.